src/Pages/instructor/manage-class/manage-class.js

```diff
@@ -3,19 +3,150 @@ import styled from 'styled-components';
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+    justify-content: space-between;
+`
+
+const StyledTableHeader = styled.div`
+    display: flex;
+    width: 100%;
+    justify-content: space-evenly;
+
+    h4 {
+        width: 20%;
+        text-align: center;
+    }
+`
+
+
+const initialSessions = [
+    {
+        name: "Boxing in an Abandoned Ring With Chris (creepy)",
+        type: "Boxing",
+        url: "https://image.shutterstock.com/image-photo/sportsman-muay-thai-boxer-fighting-260nw-626425517.jpg",
+        active: true,
+    },
+    {
+        name: "Yoga in an Abandoned Ring With Chris (creepy)",
+        type: 'Yoga',
+        url: "https://www.history.com/.image/ar_1:1%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_1200/MTY0OTE4ODAzMDQwMzE0Njg1/yoga-gettyimages-1142820079-promo.jpg",
+        active: false,
+    },
+]
+
+const initialEnrollment = [
+    {
+        time: '10/20/20 15:00',
+        duration: '50 min',
+        name: 'Boxing in an Abandoned Ring with Chris', 
+        maxsize: 20,
+    },
+    {
+        time: '10/20/20 15:00',
+        duration: '50 min',
+        name: 'Boxing in an Abandoned Ring with Chris',
+        maxsize: 20,
+    },
+    {
+        time: '10/20/20 15:00',
+        duration: '50 min',
+        name: 'Boxing in an Abandoned Ring with Chris',
+        maxsize: 20,
+    },
+]
 
 const ManageClass = () => {
 
+    const [activeSessions, setActiveSessions] = useState(initialSessions);
+    const [currentEnrollment, setCurrentEnrollment] = useState(initialEnrollment);
 
     return (
-        <div>
+        <StyledContainer>
+            <div className='manage-classes'>
 
-            this is a manage class form
-        </div>
+            
+            <h1>Manage Classes</h1>
+            <StyledMyClasses className='myClasses'>
+                <h2>My Classes</h2>
+                <div className='classList'>
+                     
+                    {
+                        activeSessions.map(session => {
+                            return <ClassCard
+                                session={session}
+                            />
+                        })
+                    }
+                    
+                    </div>
+                    
+            </StyledMyClasses>
+
+                <StyledEnrollment className='enrollment'>
+                <h2>Current Class Enrollment</h2>
+                    <StyledTable className='table'>
+                        <StyledTableHeader className='tableHeader'>
+                            <h4>Time</h4>
+                            <h4>Duration</h4>
+                            <h4>Name</h4>
+                            <h4>Size</h4>
+                            <h4></h4>
+
+                        </StyledTableHeader>
+                        {
+                            currentEnrollment.map(session => {
+                                return <ClassRow
```

Comment:
each ClassRow should have a unique key, 
`<ClassRow key={getSomeUniqueKey} session={session} />